app/src/lib/databases/repositories-database.ts

```diff
@@ -131,6 +133,13 @@ export class RepositoriesDatabase extends BaseDatabase {
 
     this.conditionalVersion(8, {}, ensureNoUndefinedParentID)
     this.conditionalVersion(9, { owners: '++id, &key' }, createOwnerKey)
+
+    this.conditionalVersion(10, {}, async tx => {
+      /**
+       * We're introducing the `organization` property on PRs in version 10.
+       */
+      tx.table('gitHubRepositories').clear()
```

Comment:
I am not 100% sure this is all I need since now the repository tables will reference a GitHub table id that doesn't exist. During development, I saw weird behavior of repositories disappearing from the repo list.. but I also had manually cleared the db.. so going to do some more testing on that... But, if you happen to know that I need more feel free to let me know. :D

Reply:
I think we shouldn't do this, it definitely sounds like it'd break things as you describe 😕 

Instead, the `organization` field for existing records in the  table `gitHubRepositories` should be `null` (or `undefined`, whatever it is the default), and then we should update the info when we have a chance.

Reply:
Yeah. Seems updating the hash is all I needed. :) It seems there is a little bit of lag in the db updating to it taking effect in the markdown. But, I think it will just be a one time blip that won't really be noticed.